src/engine/SCons/Platform/__init__.py

```diff
@@ -101,13 +101,8 @@ def platform_module(name = platform_default()):
             eval(full_name)
         else:
             try:
-                file, path, desc = imp.find_module(name,
-                                        sys.modules['SCons.Platform'].__path__)
-                try:
-                    mod = imp.load_module(full_name, file, path, desc)
-                finally:
-                    if file:
-                        file.close()
+                # the specific platform module is a relative import
+                mod = importlib.import_module("." + name, __name__)
```

Comment:
Does this imply that SCons/Platform/MY_PLATFORM in python path, but not in the scons tree wouldn't be found?  (would that have worked before?)

Reply:
Not sure. In the original form, it finds the platform file wherever it fould SCons.Platform.  Is that likely to be different than the place we're running from? Which is SCons.Platform.__init__.py ....? If so, then there's a potential variance.